design/pipeline-artifacts.md

```diff
@@ -47,9 +47,16 @@ Artifacts are downloaded either to `$PIPELINE_RESOURCESDIRECTORY` or to the dire
 
 ```yaml
 - download: string # identifier for the pipeline resource from which to download artifacts, optional; "current" means the current pipeline
```

Comment:
I think the pipeline alias `current` makes sense. This is a good time to say, resource aliases must be unique. And we should add the validation along with this feature. This gives us the flexibility in the future to expose resources as one big dictionary (not necessarily in yaml, but in expression context).

This is at least a good time to ask the question. I'm a little bit torn whether we need to do this, but leaning in the direction of buying us future flexibility.

Thoughts?

Reply:
After our conversations in December, I tend to agree. I think Patrick had some concerns (not sure what they were), did you and he get on the same page?

Reply:
👍